src/ducks/Studio/Chart/MetricsExplanation/MetricInsight/MetricInsights.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Query } from 'react-apollo'
+import Loader from '@santiment-network/ui/Loader/Loader'
+import { INSIGHT_BY_ID_QUERY } from '../../../../../queries/InsightsGQL'
+import MetricInsightCard from './MetricInsightCard'
+
+import styles from './MetricInsights.module.scss'
+
+const MetricInsights = ({ insights = [] }) => {
+  if (!insights.length) {
+    return null
+  }
+
+  return (
+    <>
+      <div className={styles.usecases}>Use cases from Insights</div>
+      <div className={styles.container}>
+        {insights.map(id => {
+          return (
+            <Query query={INSIGHT_BY_ID_QUERY} variables={{ id }} key={id}>
```

Comment:
Why do we need to load anything? @GarageInc

Reply:
@Partysun , how will we know about author and name of insight without loading? Or does it need to hardcode such data? (user can change title of insight or his username)

Reply:
@GarageInc A. Ok. Let's fetch this info.